blazor/input-mask/code-snippet/mask-configuration/EnableLiterals.razor

```diff
@@ -0,0 +1,21 @@
+﻿@using Syncfusion.Blazor.Inputs
+@using Syncfusion.Blazor.Buttons
+
+<div class="Mask-TextBox">
```

Comment:
class name should be small case. Also meaning full. Don't mention component name